### PR TITLE
Enable explore link in UN SDG tracker app

### PR DESCRIPTION
**Pull request by shifucun**

### CODE REVIEW

File: experimental/sdg-tracker/src/components/countries/CountriesContent.tsx [hunk 1]
```diff
@@ -512,10 +512,7 @@ const ChartGoalBlock: React.FC<{
   return (
     <>
       {placeDcids[0] === EARTH_PLACE_DCID && (
-        <GoalOverview
-          goalNumber={Number(goal)}
-          showExploreLink={false}
-        />
+        <GoalOverview goalNumber={Number(goal)} showExploreLink={false} />
```
Comment (pradh) [inline experimental/sdg-tracker/src/components/countries/CountriesContent.tsx:515]: meant to be false?

**Review by pradh — APPROVED** [review]

**Review by jehangiramjad — APPROVED** [review]

File: experimental/sdg-tracker/src/components/countries/CountriesContent.tsx (hunk 1 above)
Comment (shifucun) [reply experimental/sdg-tracker/src/components/countries/CountriesContent.tsx:515]: I think this is a different explore link, not related to the chart tile. I will leave it untouched.

Comment (pradh) [reply experimental/sdg-tracker/src/components/countries/CountriesContent.tsx:515]: sg

Comment (juliawu) [reply experimental/sdg-tracker/src/components/countries/CountriesContent.tsx:515]: False is correct. it should only be true if we're on the "all goals for the world" page.

**Pull request merged**